openlibrary/plugins/upstream/models.py

```diff
@@ -768,6 +772,46 @@ def as_fake_solr_record(self):
             record['subtitle'] = self.subtitle
         return record
 
+    def get_identifiers(self):
+        """Returns (name, value) pairs of all available identifiers."""
+        names = []
+        return self._process_identifiers(
+            get_work_config().identifiers, names, self.identifiers
+        )
+
+    def _process_identifiers(self, config_, names, values):
```

Comment:
Is this method the same for editions? If so I think we might want to create a helper method at the bottom of this file that both classes call, to avoid duplicating the code, if possible!

Reply:
https://github.com/internetarchive/openlibrary/blob/7a4fec1cb44829d421764baf0c65d2993a942455/openlibrary/plugins/upstream/utils.py#L1171-L1202

Is this something close to what you had in mind? I noticed that the author identifier config method followed a similar pattern to the work config method as well